docker-compose.yml

```diff
@@ -21,17 +21,17 @@ services:
           "--layer-average-size", "${SMESH_LAYER_SIZE:-200}",
           "--hare-committee-size", "${SMESH_HARE_COMMITTEE_SIZE:-800}",
           "--hare-max-adversaries", "${SMESH_HARE_MAX_ADVERSARIES:-399}",
-          "--tortoise-beacon-kappa", "${SMESH_TORTOISE_BEACON_KAPPA:-40}",
-          "--tortoise-beacon-q", "${SMESH_TORTOISE_BEACON_Q:?1/3}",
-          "--tortoise-beacon-rounds-number", "${SMESH_TORTOISE_BEACON_ROUNDS_NUMBER:-300}",
-          "--tortoise-beacon-grace-period-duration", "${SMESH_TORTOISE_BEACON_GRACE_PERIOD_DURATION:?2m}",
-          "--tortoise-beacon-proposal-duration", "${SMESH_TORTOISE_BEACON_PROPOSAL_DURATION:?2m}",
-          "--tortoise-beacon-first-voting-round-duration", "${SMESH_TORTOISE_BEACON_FIRST_VOTING_ROUND_DURATION:?1h}",
-          "--tortoise-beacon-voting-round-duration", "${SMESH_TORTOISE_BEACON_VOTING_ROUND_DURATION:?30m}",
-          "--tortoise-beacon-weak-coin-round-duration", "${SMESH_TORTOISE_BEACON_WEAK_COIN_ROUND_DURATION:?1m}",
-          "--tortoise-beacon-theta", "${SMESH_TORTOISE_BEACON_THETA:?1/4}",
-          "--tortoise-beacon-votes-limit", "${SMESH_TORTOISE_BEACON_VOTES_LIMIT:-100}",
-          "--tortoise-beacon-sync-num-blocks", "${SMESH_TORTOISE_BEACON_SYNC_NUM_BLOCKS:-1600}",
+          "--beacon-kappa", "${SMESH_TORTOISE_BEACON_KAPPA:-40}",
```

Comment:
`SMESH_TORTOISE_BEACON_KAPPA` and others still contain tortoise beacon, not just beacon. Also in `.env`

Reply:
done. thanks